cloud/app/middleware.py

```
@@ -0,0 +1,31 @@
+def setup_middleware(app: FastAPI, settings: Settings) -> None:
+    """Configure middleware stack."""
+    app.add_middleware(
+        CORSMiddleware,
+        allow_origins=settings.cors_origins,
+        allow_credentials=True,
+        allow_methods=["*"],
+        allow_headers=["*"],
+    )
+
+    @app.middleware("http")
+    async def log_requests(request: Request, call_next):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `log_requests`

(ANN202)